pyproject.toml

```diff
@@ -134,7 +134,6 @@ disable = [
   "invalid-envvar-default",
   "invalid-overridden-method",
   "keyword-arg-before-vararg",
-  "method-cache-max-size-none",
```

Comment:
@MarcoGorelli have we been using pylint since releasing 1.5? Not sure if this change will cause a merge conflict in backported, and I dont think it's worth backporting pylint configurations to 1.5.x

Reply:
agree about not backporting the configuration, I'd just backport the fix itself - I'd expect to get a merge conflict, I'll sort this out there